docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -280,6 +289,64 @@ similar to `Environment`.
 
 This key can be listed multiple times.
 
+### Kube units
+
+Kube units are named with a `.kube` extension and contain a `[Kube] `section describing
+how `podman kube play` should be run as a service. The resulting service file will contain a line like
+`ExecStart=podman kube play … file.yml`, and most of the keys in this section control the command-line
+options passed to Podman. However, some options also affect the details of how systemd is set up to run and
+interact with the container.
+
+There is only one required key, `Yaml`, which defines the path to the Kubernetes YAML file.
+
+Supported keys in `Container` section are:
```

Comment:
```suggestion
Supported keys in the `Container` section are:
```

Reply:
Thanks, this one was even worse because the section's name is `Kube`. Anyhow, it's fixed now